go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.2.1
+	github.com/pseudomuto/protoc-gen-doc v1.3.2 // indirect
```

Comment:
`go mod tidy` invocation removes this indirect dependency.

Reply:
I'll explore where this is used. I'll make sure if it is not part of the make release target it gets added in a future PR

Reply:
I will either add this to a future PR for either the Release process or make release target. Could not locate any use of it.

Reply:
Actually I found where it is used. 

https://github.com/googleforgames/open-match/blob/4b3147511bdcc292b711390532fd10ea6a5da8eb/Makefile#L696

Reply:
This looks to be used for generating documentation for the docs repository

Reply:
Seems that without go.mod changes should work with this make command:
https://github.com/googleforgames/open-match/blob/fcd590eca689311fa31b5a68c0935c2b11aa913a/Makefile#L561

Reply:
I see. I will submit a PR with the changes.